docker-compose.yaml

```diff
@@ -229,6 +229,7 @@ services:
     user: "0:0"
     volumes:
       - ${AIRFLOW_PROJ_DIR}:/sources
+      - ${AIRFLOW_PROJ_DIR}/airflow.cfg:/opt/airflow/airflow.cfg
```

Comment:
@mddilley @frankhereford here was the problem. even though this service uses the `airflow-common-env` - the `volumes` setting gets overwritten here. so the init script was attempting to fire with zero configuration :/